common/src/main/java/com/wynntils/models/gear/GearChatEncoding.java

```diff
@@ -201,7 +201,9 @@ public GearItem fromEncodedString(String encoded) {
         }
 
         // create chat gear stack
-        GearInstance gearInstance = GearInstance.create(gearInfo, identifications, powderList, rerolls);
+        // TODO: 8/7/2023 replace Optional.empty() with actual shiny mythic stat when it exists
```

Comment:
That is never going to happen. We cannot change the "chat transfer" protocol to support new functions, due to the lack of version handling in the protocol. :-(

Reply:
Or rather, it basically requires us to design a new protocol. And figure out how to handle old and incompatible clients.

Reply:
I mean my first instinct is to suggest attaching the shiny stat to the the end of the existing item string with a spacebar in-between so older clients will just parse the item the same but updated clients can see the stat by checking if there's more unicode encoded item info after the space char.

(but would that be a janky solution idk.)

Reply:
The real solution is that we need to create a new "protocol" that has some kind of version information, or is extendable in some other way, so we can continue adapting for future Wynn changes. But that is a quest for another day...

Reply:
following through that idea, maybe encoding the statistic name with a single byte should be fine I doubt I'll see the day in which wynncraft has 255 or something statistics on shiny mythics, but who knows, if we suppose that wynncraft will eventually have the 128th unique shiny mythic statistic, we can reserve the first bit in that as a flag value just in case this happens and if that bit is true we can check the next byte as well or something. (can't wait for the day when wynncraft releases the 128th unique shiny mythic stat, and we don't have to worry about backwards compatibility in that very specific scenario.

(but haven't I made a janky enough of an idea maybe shouldn't make it jankier.

Reply:
Please just remove this comment, and then you're good to go.

Reply:
tldr: <itemname> + ' ' + new stuff here

Reply:
Problem is, there are already other information we can't properly pass using this chat protocol. So we need to fix that as well.

Reply:
oh, that makes sense.